pkg/component/gardener/controllermanager/configmaps.go

```diff
@@ -66,6 +66,9 @@ func (g *gardenerControllerManager) configMapControllerManagerConfig() (*corev1.
 			SecretBinding: &controllermanagerv1alpha1.SecretBindingControllerConfiguration{
 				ConcurrentSyncs: ptr.To(20),
 			},
+			CredentialsBinding: &controllermanagerv1alpha1.CredentialsBindingControllerConfiguration{
+				ConcurrentSyncs: ptr.To(20),
+			},
```

Comment:
Add this to https://github.com/gardener/gardener/tree/master/charts/gardener/controlplane as well